packages/@react-aria/virtualizer/src/Virtualizer.tsx

```diff
@@ -118,12 +122,13 @@ export function useVirtualizer<T extends object, V, W>(props: VirtualizerOptions
       if (scrollToItem) {
         scrollToItem(focusedKey);
       } else {
-        virtualizer.scrollToItem(focusedKey, {duration: 0});
+        // offsetY required to have android talkback accessibility cursor properly positioned after combobox tray menu is scrolled
+        virtualizer.scrollToItem(focusedKey, {duration: 0, offsetY: ref.current.offsetTop});
```

Comment:
@LFDanLu Could you give me a bit more info here? I'm not quite following how this makes sense. `ref.current.offsetTop` would be the position of the listbox itself, but you're scrolling inside the listbox so that should have nothing to do with the scroll position... Do you remember?

Reply:
urgh, if I remember correctly this was an attempt to fix an issue with the talkback cursor where it would jump too far down when using the swipe gestures (I'll send you a screen recording). I don't quite recall why I used `offsetTop` here but it seemed to fix to problem at the time but doesn't now (yeah I dunno...). I'll remove it and try a different approach.